include/z64animation.h

```diff
@@ -99,18 +99,18 @@ typedef struct {
 } LegacyAnimationHeader; // size = 0xC
 
 typedef enum {
-    ANIMATION_LINKANIMETION,
-    ANIMENTRY_COPYALL,
-    ANIMENTRY_INTERP,
-    ANIMENTRY_COPYTRUE,
-    ANIMENTRY_COPYFALSE,
-    ANIMENTRY_MOVEACTOR
+    /* 0 */ ANIMATION_LINKANIMETION,
```

Comment:
Should this be `PLAYERANIMETION`?

Reply:
Good question. 
I didn't rename it because I thought it was named after the `link_animetion` segment (notice it says "animetion" instead of "animation"). If that's the case I think we should rename this whenever we rename said segment